<ui version="4.0" >
 <class>Info</class>
 <widget class="QWidget" name="Info" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Info</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Now Playing</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <item>
        <widget class="QToolBar" name="col_toolbar" >
         <addaction name="separator" />
         <addaction name="actionBan" />
         <addaction name="actionRateDown" />
         <addaction name="actionRateUp" />
         <addaction name="actionLoveIt" />
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_0" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <property name="leftMargin" >
                <number>4</number>
               </property>
               <property name="topMargin" >
                <number>4</number>
               </property>
               <property name="rightMargin" >
                <number>4</number>
               </property>
               <property name="bottomMargin" >
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_so1" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text" >
                  <string>Song Title</string>
                 </property>
                 <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="StarEditor" native="1" name="songRating" />
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <property name="leftMargin" >
                <number>0</number>
               </property>
               <property name="topMargin" >
                <number>0</number>
               </property>
               <property name="rightMargin" >
                <number>0</number>
               </property>
               <property name="bottomMargin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_al0" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>65535</width>
                   <height>65535</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="pixmap" >
                  <pixmap resource="resource.qrc" >:/icons/def_album.png</pixmap>
                 </property>
                 <property name="scaledContents" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_5" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow" >
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" >
                  <property name="spacing" >
                   <number>0</number>
                  </property>
                  <property name="leftMargin" >
                   <number>4</number>
                  </property>
                  <property name="topMargin" >
                   <number>4</number>
                  </property>
                  <property name="rightMargin" >
                   <number>4</number>
                  </property>
                  <property name="bottomMargin" >
                   <number>4</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_al1" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text" >
                     <string>Album Title</string>
                    </property>
                    <property name="alignment" >
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="StarEditor" native="1" name="albumRating" />
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Artist</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit" />
       </item>
       <item>
        <widget class="QListWidget" name="listWidget" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Album</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>4</number>
       </property>
       <property name="topMargin" >
        <number>4</number>
       </property>
       <property name="rightMargin" >
        <number>4</number>
       </property>
       <property name="bottomMargin" >
        <number>4</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit_2" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionRateUp" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/star.png</iconset>
   </property>
   <property name="text" >
    <string>RateUp</string>
   </property>
   <property name="toolTip" >
    <string>Rate Up</string>
   </property>
  </action>
  <action name="actionRateDown" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/antistar.png</iconset>
   </property>
   <property name="text" >
    <string>RateDown</string>
   </property>
   <property name="toolTip" >
    <string>Rate Down</string>
   </property>
  </action>
  <action name="actionLoveIt" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/heart.png</iconset>
   </property>
   <property name="text" >
    <string>LoveIt</string>
   </property>
  </action>
  <action name="actionBan" >
   <property name="icon" >
    <iconset resource="resource.qrc" >:/icons/deltab.png</iconset>
   </property>
   <property name="text" >
    <string>Ban</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StarEditor</class>
   <extends>QWidget</extends>
   <header>stareditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>Info</receiver>
   <slot>tabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>161</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBan</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_ban()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoveIt</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_loveIt()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRateDown</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_rateDown()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRateUp</sender>
   <signal>triggered()</signal>
   <receiver>Info</receiver>
   <slot>slot_rateUp()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>112</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
